src/boot/efi/boot.c

```diff
@@ -821,11 +818,7 @@ static BOOLEAN menu_run(
                         break;
 
                 case KEYPRESS(0, 0, 'v'):
-                        status = PoolPrint(L"systemd-boot " GIT_VERSION " (" EFI_MACHINE_TYPE_NAME "), UEFI Specification %d.%02d, Vendor %s %d.%02d",
-                                           ST->Hdr.Revision >> 16, ST->Hdr.Revision & 0xffff,
-                                           ST->FirmwareVendor, ST->FirmwareRevision >> 16, ST->FirmwareRevision & 0xffff);
-                        break;
```

Comment:
this is useful too, don't drop.

i mean, it's fine to remove them from the help text + man page, since they are kinda developer features, but i'd really keep them in the code, and just add a comment there that they are a developer feature and not documented on purpose.

Reply:
This is in print_status() and I reassigned v to that. Shouldn't that be enough?

Reply:
i'd keep it. it doesn't hurt, and i actually use it myself, to see if my images are built correctly.